Week-5/exercise-5.2/vowel-count.js

```diff
@@ -0,0 +1,19 @@
+/*
+    Write a function called vowelCount which accepts a string and
+    returns a map where the keys are vowels and the values are the count of the vowels in the string
+*/
+var log4js = require("log4js");
+var logger = log4js.getLogger();
+
+const vowels = { a: 0, e: 0, i: 0, o: 0, u: 0 };
+const vowel_count = (name) => {
+  const mName = name.toLowerCase();
+  let map = new Map(Object.entries(vowels));
```

Comment:
you don't need to use Object.entries to initialize the map with default value

could have used 

new Map([
        ['a',0],
        ['e',0],
        ['i',0],
        ['o',0],
        ['u',0]
    ])